.azure-pipelines/templates/automation_test.yml

```diff
@@ -37,16 +37,16 @@ steps:
       module="${{ parameters.module }}"
       if [[ -n $module ]]; then
         echo "Running test for module '$module'"
-        azdev test --no-exitfirst --verbose --series $module
+        azdev test --no-exitfirst --verbose --series $module -a "-n 4 --durations=10"
```

Comment:
`-n 4` means 4 tests can run concurrently, but many Azure CLI tests cannot be run concurrently. That's why we have that `--series` flag.